app/helpers/utils.py

```diff
@@ -32,7 +35,10 @@ def store_results(data):
     result.save()
 
 def has_access_to_project(project_id,user_id):
-    return ProjectModel.is_a_member_of_project(project_id,user_id)
+    if project_id is None or user_id is None:
+        return False
+    else:
+        return ProjectModel.is_a_member_of_project(project_id,user_id)
```

Comment:
I think if the condition is true function returns, then we can reduce if-else ladder by avoiding else.
```suggestion
    if project_id is None or user_id is None:
        return False
    return ProjectModel.is_a_member_of_project(project_id,user_id)
```